pglookout/pglookout.py

```diff
@@ -174,6 +174,12 @@ def load_config(self, _signal=None, _frame=None):
                 self.replication_lag_warning_boundary = self.replication_lag_failover_timeout
                 msg = "Replication lag warning boundary set to %s"
                 self.log.warning(msg, self.replication_lag_warning_boundary)
+
+        # regenerate recovery-config if we're following somebody else, as the connection info could have changed
+        if self.current_master and self.own_db and self.own_db != self.current_master:
+            # this is actually a no-op if the config-file doesn't change
+            self.start_following_new_master(self.current_master)
```

Comment:
I think this actual following should be done in a main loop somewhere, so that in case of temporary error (new master is not responding or something) we don't crash here.

Reply:
I've extracted it into a separate function that is triggered by the main loop, if the configuration got reloaded.